### ISSUE TITLE
I can't choose the language option

### ISSUE THREAD

**Issue by hlj2722** [ASKBOT/askbot-devel#547]
I enable the  askbot.middleware.locale.LocaleMiddleware ,But there is no Chinese language option in the Settings page(Settings → Static Content, URLS & UI → Skin, logos and HTML parts ). There Select Lanuage option is only English,not other option.What's the problem?

**Comment by mkissam** [comment]
What are your Language settings? LANGUAGE_CODE, ASKBOT_LANGUAGE_MODE, LANGUAGES ?

**Comment by hlj2722** [comment]
LANGUAGES = (('en', 'English'),('de','German'),('zh-cn','Simplified Chinese'))

This is my language settings

**Comment by parrah** [comment]
Have you tried changing ASKBOT_LANGUAGE_MODE to 'url-lang'?

**Comment by mkissam** [comment]
Yeah, so the ASKBOT_LANGUAGE_MODE = 'single-lang' definitely a problem. Try the url-lang setting suggested above.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.

**Comment by y00251390** [comment]
I met the same problem,When I set url-lang,it no works.
